lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -935,9 +960,13 @@ def corrupt_agw_state(self, key: str):
 
     def restart_all_services(self):
         """Restart all magma services on magma_dev VM"""
-        self.exec_command(
-            "sudo service magma@* stop ; sudo service magma@magmad start",
-        )
+        if self._init_system == InitMode.SYSTEMD:
+            self.exec_command(
+                "sudo service magma@* stop ; sudo service magma@magmad start",
+            )
+        elif self._init_system == InitMode.DOCKER:
+            self.exec_command("cd /home/vagrant/magma/lte/gateway/docker")
```

Comment:
I am surprised that this works, I would expect this to translate to two ssh commands. How does the second command know about the `cd`?

Reply:
You're right, thanks for spotting this! I've combined the two lines into one command.